File: 01-arrays-and-strings/reverse.py
#Write a method/function, reverse(A) that takes an array of integers, A and reverses it

def reverse(A):
    size = len(A)
    result = []
    for x in range(1, size + 1):
        result.append(A[-x]) #Used to append values starting from the last integer in the list
    return result

#Testing purposes
A = [3,2,1,0]
print(reverse(A))

File: classwork/reverse.js
function reverse(A) {
    let reversedA = []
    for (let i = A.length - 1; i >= 0; i--) {
       reversedA.push(A[i]);
    }
    return reversedA;
}

let a = [1, 4, 5, 6];
console.log(reverse(a));
